### ISSUE TITLE
Unable to control number of suggestions before introduce scrolling and the no option to clear the values on the typeahead of the dropdown

### ISSUE THREAD

**Issue by AnurudhaNagarajan** [predixdesignsystem/px-dropdown#85]
Hi, 

The search box on the typeahead of the dropdown introduces the scroll bar when the items are more than 8. Is there an option to control it. 

The autocomplete in the typeahead of the dropdown does not clear on its own.

**Comment by evanjd** [comment]
@AnurudhaNagarajan Can you provide some sample code to reproduce this? I've created a dropdown with 8+ items and I'm not seeing any typeahead or autocomplete offered.

**Comment by evanjd** [comment]
Those same settings here:

<img width="228" alt="screen shot 2018-08-09 at 8 33 15 pm" src="https://user-images.githubusercontent.com/4055224/43893787-7c7453f4-9c13-11e8-9883-03395f56a7c5.png">

What version are you using?

**Comment by AnurudhaNagarajan** [comment]
<px-dropdown id="customerDropdown" class="landing-dropdown"
        [items]="customerList"
        (px-dropdown-selection-changed) ="getModel($event)"
        (selected-changed) ="getSelected($event)"
        sort-mode="val"
        button-style="bare"
        search-mode
        display-value="Customer"
        trigger-height="30">
      </px-dropdown> 
![dropdownwithoutscroll](https://user-images.githubusercontent.com/18552273/43894017-7edbdba0-9bee-11e8-96e7-272b445790a4.PNG)
![dropdownwithscroll](https://user-images.githubusercontent.com/18552273/43894018-7f702256-9bee-11e8-9564-4f88af43a291.PNG)

**Comment by AnurudhaNagarajan** [comment]
"px-dropdown": "^4.7.7"
I wanted to set it to 10 then for the scroll to appear

**Comment by evanjd** [comment]
Again, same settings here, no scrollbar.

<img width="214" alt="screen shot 2018-08-09 at 8 40 18 pm" src="https://user-images.githubusercontent.com/4055224/43894118-7e47e32a-9c14-11e8-8e80-51d2bd754ca6.png">

Can you please provide more info - what browser and what version of px-dropdown you're using.

**Comment by AnurudhaNagarajan** [comment]
Chrome 68

**Comment by evanjd** [comment]
OK. What CSS belongs to the `landing-dropdown` class?

**Comment by AnurudhaNagarajan** [comment]
Landing dropdown is for the dropdown width, height

**Comment by AnurudhaNagarajan** [comment]
since it is placed inside a flex box

**Comment by evanjd** [comment]
I'm asking for the actual CSS that belongs to landing-dropdown so that I can replicate your conditions as precisely as possible, since I cannot currently reproduce it.

**Comment by AnurudhaNagarajan** [comment]
.landing-dropdown {
    height: 60px !important;
    padding-top: 20px;
    padding-left: 20px;  
    --px-btn-bare-color--hover:black;
    --px-dropdown-bg-color--selected:#007ACC;
}

**Comment by evanjd** [comment]
Still not reproducible. I've gone ahead and created a glitch for you: https://aspiring-penalty.glitch.me/. Does it appear correct there?

You can edit it here: https://glitch.com/edit/#!/aspiring-penalty

Also, just making sure, what theme are you using, and how are you importing it? For reference, this is how the default theme is imported.

In `<head>`: `<link rel="import" href="path/to/bower/components/px-theme/px-theme-styles.html">`

In `<body>`: `<custom-style><style include="px-theme-styles" is="custom-style"></style></custom-style>`

**Comment by AnurudhaNagarajan** [comment]
Yes, it seems to be right there. I have imported the themes. Also any idea on the autocomplete on the search box of dropdown. Users need to manually clear it which is a problem

**Comment by evanjd** [comment]
There's not much I can do unless you can demonstrate the bug in an environment I can access and view the implementation for. Feel free to remix the glitch I provided - if you can reproduce the bug there please share the link here.

On the autocomplete - there isn't one, so I'm not sure what you mean.

**Comment by AnurudhaNagarajan** [comment]
https://px-dropdown-issue.glitch.me/
